assets/translations/main.json

```diff
@@ -101,6 +101,8 @@
 	"action.zoom-to-100": "Zoom to 100%",
 	"action.zoom-to-fit": "Zoom to fit",
 	"action.zoom-to-selection": "Zoom to selection",
+	"assets.files.upload-failed": "Upload failed",
+	"assets.url.failed": "Extracting bookmark info failed",
```

Comment:
This feels a bit technical - what about "Couldn't load URL preview" or "Getting preview failed" or something like that?